README.md

```diff
@@ -53,7 +53,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+unsure, I forget the difference between = and ===
```

Comment:
See above re: `=`. 
`===` is the strict equality comparison. It takes two arguments and evaluates to either `true` or `false` based on whether or not the two arguments are exactly equal.